### PR TITLE
reorganize the struct

### PR DESCRIPTION
**Pull request by estarriolvetch**
Easy gas saving by reorganise the struct.

By reorganising the struct, it uses 3 less storage slots which will save `60000 * 2 = 120000` gas in registering

### CODE REVIEW

File: src/DelegationManagement.sol [hunk 1]
```diff
@@ -10,7 +10,7 @@
  *
  */
 
-pragma solidity >=0.6.0 <=0.8.0;
+pragma solidity >=0.8.0;
```
Comment (rookmate) [inline src/DelegationManagement.sol]: Please split the compiler change into a different commit on the same PR. Otherwise it is misleading to who is analyzing the PR

**Review by rookmate — CHANGES_REQUESTED** [review]

File: src/DelegationManagement.sol (hunk 1 above)
Comment (estarriolvetch) [reply src/DelegationManagement.sol]: sure

**Comment by a2rocket** [comment]
outdated
